webview/src/stories/Plots.stories.tsx

```diff
@@ -52,7 +53,23 @@ const MockedState: React.FC<{ data: PlotsData; children: React.ReactNode }> = ({
 
 const defaultPlotsData = {
   cliError: null,
-  comparison: comparisonPlotsFixture,
+  comparison: {
+    ...comparisonPlotsFixture,
+    plots: comparisonPlotsFixture.plots.map(plot => {
+      const classDetailsArr = Object.entries(plot.classDetails)
+
+      if (classDetailsArr.length === 0) {
+        return plot
+      }
+
+      classDetailsArr.unshift([
+        'tree',
+        { color: getBoundingBoxColor(classDetailsArr.length), selected: false }
+      ])
```

Comment:
Added a `unselected` option that we could view in Storybook since checks rely on the backend.